doc/whats_new/v0.21.rst

```diff
@@ -403,6 +403,11 @@ Support for Python 3.4 and below has been officially dropped.
 - |API| :class:`pipeline.Pipeline` now supports using ``'passthrough'`` as a
   transformer. :issue:`11144` by :user:`Thomas Fan <thomasjpfan>`.
 
+- |Enhancement| :class:`pipeline.Pipeline` now implements a func
+  :func:`__len__()`,
+  that will return the length of the pipeline :issue:`13439`
```

Comment:
Please put . after "pipeline". 

It would be nice if this message made it clearer to users that `len` will now work

Reply:
ok I will shift the . after pipeline, do you want me to modify the message and write like this: 
:class:`pipeline.Pipeline` now implements a func len, 
  that will now work and return the length of the pipeline.  :issue:`13439` by :user:`Lakshya KD <LakshKD>`.

Reply:
I have modified the message, do let me know if it requires any changes.